### ISSUE TITLE
Ability to directly reply to Talk notifications

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/notifications#637]
(Opened here instead of Talk because I assume it needs some groundwork here first, feel free to transfer the issue.)

Currently when you get a notification about a Talk message, you can only "View chat". This takes quite long and often you just want to quickly reply without taking you out of your flow.

It would be very cool to have a way to directly reply to the message, which would automatically @mention that person and append your message, without you needing to leave your place and flow. (Android has this functionality on its notifications and it’s quite useful.)

Current | Direct reply proposal
-----------| ------------
![Nextcloud Notifications current](https://user-images.githubusercontent.com/925062/81416361-e511a380-9149-11ea-84ab-54a3c9fa1348.png) | ![Nextcloud Notifications direct reply](https://user-images.githubusercontent.com/925062/81416354-e347e000-9149-11ea-978c-2e23bee61c27.png)

cc @nickvergessen @ma12-co

**Comment by wiswedel** [comment]
Great idea. I suggest to keep the "view chat" opportunity also. You might wanna check the context first before replying.

**Comment by jancborchardt** [comment]
Yup, we can keep "View chat" and add a primary button "Reply" which opens and focuses the input field.

**Comment by ma12-co** [comment]
Nice :) 
I'd suggest to make the whole notification clickable / tappable to go to the chat instead of using a button.

**Comment by wiswedel** [comment]
> instead of using a button.

That would indeed be more mobile-like, but I guess disqualifying for accessibility and explicity reasons, right @jancborchardt ?

**Comment by ma12-co** [comment]
I don't think so, at least with proper hover feedback and cursor: pointer. 
E.g. Twitter notifications: 

![Peek 2020-05-13 09-47](https://user-images.githubusercontent.com/26852655/81785680-db4bbf80-94fe-11ea-87d2-9604bee6d423.gif)

**Comment by jancborchardt** [comment]
> I'd suggest to make the whole notification clickable / tappable to go to the chat instead of using a button.

Sure thing, but separate issue. :) This is about the "Ability to directly reply to Talk notifications". ;)

**Comment by jancborchardt** [comment]
This is now in the works on the desktop client, would be nice to get feature parity to release this in concert. Is this something to be done as part of Talk team as set in the project @nickvergessen, or does it have to do with server/files too @PVince81?

**Comment by nickvergessen** [comment]
Well either team can. Either talk because it's talk, or server/files because that is where the app lives.
